packages/check-bundle/lite.js

```diff
@@ -29,30 +29,28 @@ export const checkBundle = async (
       bundle,
     )}`,
   );
-  assert(bundle !== null, d`checkBundle expects a bundle object`);
-  assert(
-    Object.isFrozen(bundle),
-    `checkBundle cannot vouch for the ongoing integrity of an unfrozen object, got ${q(
+  if (bundle === null) {
+    throw Fail`checkBundle expects a bundle object`;
+  }
+  Object.isFrozen(bundle) ||
+    Fail`checkBundle cannot vouch for the ongoing integrity of an unfrozen object, got ${q(
       bundle,
-    )}`,
-  );
+    )}`;
   const properties = Object.entries(Object.getOwnPropertyDescriptors(bundle));
   const nonValues = properties.filter(
     ([, property]) => typeof property.get === 'function',
   );
   const nonStrings = properties.filter(
     ([, property]) => typeof property.value !== 'string',
   );
-  assert(
-    nonValues.length === 0 && nonStrings.length === 0,
-    `checkBundle cannot vouch for the ongoing integrity of a bundle ${q(
+  (nonValues.length === 0 && nonStrings.length === 0) ||
+    Fail`checkBundle cannot vouch for the ongoing integrity of a bundle ${q(
```

Comment:
Reviewers, note that the original did not say `X` and some of the substitutions do not have `q`, so this change is not equivalence preserving. Is this a problem? Should the remaining substitutions be unredacted with `q`?

What the correct answer is depends on the intention of the code. Please look for this possible issue everywhere the original did not have an `X` and advise. Thanks.

Reply:
Should we update `assert` to detect when the output from `quote` is serialized by anything other than `details` and surface the problem by something like `console.error` and/or creating an unhandled rejection and/or forcing a nonzero process exit status?

Reply:
I don't understand the question. Let's be sure to discuss in our upcoming meeting.

Reply:
Oops. We forgot to discuss this. Could you expand or clarify a bit here?

Reply:
I was brainstorming ways to detect and address accidental omission of `X` in assertion arguments, but the real risk is placeholder expressions that _don't_ use `q`/`assert.quote`, and it's now basically moot anyway because we no longer encourage or expect direct use of `X`/`assert.details` to construct arguments. So I consider this resolved.